src/sections/ui/tooltip/QuestionIcon.module.scss

```diff
@@ -0,0 +1,10 @@
+$fill-color: #99bcdb;
+$hover-color: #337ab7;
```

Comment:
Should we add these colors as part of the design tokens of the theme? What do you think? If people is able to change the primary and secondary color maybe this blue color for the tooltips doesn't look good in some customizations 

For adding these to the theme we need to:

1. Add these variables to the /ui/assets/styles/design-tokens/colors.scss and add them to the `export: {}`
2. Add these variables to the /ui/theme/basicTheme.ts object
3. Update Colors.mdx stories to register the tooltips colors

Reply:
yes, I think that's a good idea, just committed a change for that.

Reply:
I added the colors to the theme as Melina suggested.